docs/.vitepress/theme/components/home/HomeIconCustomizer.vue

```diff
@@ -48,10 +48,16 @@ function resetStyle () {
   color.value = 'currentColor'
   strokeWidth.value = 2
   size.value = 24
+  absoluteStrokeWidth.value = false;
+  nextTick(() => {
+    
+  });
+
```

Comment:
Should this not be in the nextTick function?
```suggestion
  nextTick(() => {
    absoluteStrokeWidth.value = false;
  });

```

Reply:
On further reflection I think I may not have needed to include nextTick in the first place. Sorry this is my first time using Vue. I updated the branch accordingly and again thank you for helping me on this.